js/app/controllers/attendeecontroller.js

```diff
@@ -62,7 +62,8 @@ app.controller('AttendeeController', function($scope, AutoCompletionService) {
 					'role': 'REQ-PARTICIPANT',
 					'rsvp': 'TRUE',
 					'partstat': 'NEEDS-ACTION',
-					'cutype': 'INDIVIDUAL'
+					'cutype': 'INDIVIDUAL',
+					'language': OC.getLocale() // Use current user's timezone as a default value
```

Comment:
Instead of OC.getLocale() for the default value the default should be the value of the user value 'core' 'lang'.  When loading the page this can be added to the parameters by retrieving  $this->config->getUserValue($user->getUID(), 'core', 'lang'); in viewcontroller.php.